conans/client/cmake.py

```diff
@@ -330,6 +340,11 @@ def _build_new(self, args=None, build_dir=None, target=None):
         command = "cmake --build %s" % arg_list
         self._conanfile.run(command)
 
+    def package(self):
```

Comment:
It is the equivalent of the ``package()`` method. It is using the ``install`` feature of cmake to implement the ``package()`` functionality. It will not install in the system, it will create the conan package in the package_folder.